src/components/templates/ImportFileZone.jsx

```diff
@@ -209,9 +210,11 @@ class ImportFileZone extends Component {
     return (
       <section>
         <div className="ImportFileZone" style={importFileZone}>
+          <FontAwesomeIcon icon={faUpload} />
```

Comment:
Sorry to be fiddly, but can you adjust the icon alignment?
<img width="311" alt="image" src="https://user-images.githubusercontent.com/588335/69265546-49ca4900-0b7e-11ea-9237-3575c1b0f2df.png">

Reply:
yeah, that bugged me too, but i had a hard time doing it in CSS...easy fix is to make it part of the link, but that makes the icon blue and underlines it like this:

![Screen Shot 2019-11-20 at 10 24 02 AM](https://user-images.githubusercontent.com/47137/69266373-e6d9b180-0b7f-11ea-9266-fb991bfa2364.png)

Reply:
never mind - aligned correctly